app/packages/components/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -66,6 +71,9 @@ const Errors = (onReset?: () => void, disableReset?: boolean) => {
       resetErrorBoundary();
     }
 
+    console.log({ error });
+    console.log(error.stack);
```

Comment:
Is this intentional logging? Maybe `console.error` should be used?

Reply:
Leftover. Will remove